project-management/src/main/java/life/qbic/projectmanagement/application/api/AsyncProjectService.java

```diff
@@ -37,6 +37,10 @@
  */
 public interface AsyncProjectService {
 
+  /*
+  API method section - start
+   */
+
```

Comment:
What is this?

Reply:
Helps structuring the class into API methods and concepts